parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -266,6 +272,43 @@ private WriteBuilder createContextFunc(
       return this;
     }
 
+    private <T> void setBloomFilterConfig(
+        Context context,
+        MessageType parquetSchema,
+        BiConsumer<String, Boolean> withBloomFilterEnabled,
+        BiConsumer<String, Double> withBloomFilterFPP) {
+
+      Map<Integer, String> fieldIdToParquetPath =
+          parquetSchema.getColumns().stream()
+              .collect(
+                  Collectors.toMap(
+                      col -> col.getPrimitiveType().getId().intValue(),
+                      col -> String.join(".", col.getPath())));
+
+      context
+          .columnBloomFilterEnabled()
+          .forEach(
+              (colPath, isEnabled) -> {
+                Types.NestedField fieldId = schema.caseInsensitiveFindField(colPath);
```

Comment:
[doubt] does sensitivity matters ? can this : 

```
write.parquet.bloom-filter-enabled.column.CoL1
```

be applied to parquet files with schema containing `col1` ?

if not should we explicitly do lowercase post deriving the configs

Reply:
Sensitivity matters. I changed to `findField`. Thanks!